addons/portal/controllers/portal.py

```diff
@@ -308,10 +308,65 @@ def account(self, redirect=None, **post):
     def on_account_update(self, values, partner):
         pass
 
+    def _get_commercial_partners_and_users(self):
+        partner = request.env.user.partner_id
+        commercial_partners = partner.commercial_partner_id.child_ids
+        portal_users = commercial_partners.filtered(lambda partner: partner.user_ids and partner.user_ids[0].has_group('base.group_portal')) \
+                                          .mapped(lambda partner: partner.user_ids[0] if partner.user_ids else partner.user_ids)
+        return commercial_partners, portal_users
+
+    def _prepare_users_accesses_values(self):
+        values = self._prepare_portal_layout_values()
+        values.update({
+            'page_name': 'users_accesses',
+            'get_message': get_message,
+        })
+        return values
+
+    @route('/my/users_accesses', type='http', auth='user', website=True, methods=['POST'])
```

Comment:
Strange naming ... ``/my/company_access`` maybe ?

Reply:
Would you use this wording for displayed text too ? (the menu is called Users Accesses right now)